modules/hostap/src/supp_api.c

```diff
@@ -1128,6 +1128,50 @@ int supplicant_pmksa_flush(const struct device *dev)
 	return ret;
 }
 
+int supplicant_11k_cfg(const struct device *dev, struct wifi_11k_params *params)
+{
+	const struct wifi_mgmt_ops *const wifi_mgmt_api = get_wifi_mgmt_api(dev);
+
+	if (!wifi_mgmt_api || !wifi_mgmt_api->cfg_11k) {
+		wpa_printf(MSG_ERROR, "cfg 11k not supported");
+		return -ENOTSUP;
+	}
+
+	return wifi_mgmt_api->cfg_11k(dev, params);
+}
+
+int supplicant_11k_neighbor_request(const struct device *dev, struct wifi_11k_params *params)
+{
+	int ssid_len = strlen(params->ssid);
+
+	if (params && ssid_len) {
```

Comment:
There has been some effort in the past to write the non boolean type checks non boolean way. So what you have here works, it would be preferable to write this as
```
if (params != NULL && ssid_len > 0) {
```

Reply:
Updated.